scripts/deploy-ncp-pull.sh

```
@@ -1,350 +1,375 @@
 }
 
-if [[ "${DEPLOY_MODE}" == "rollback" ]]; then
-  require_command docker
-  require_command curl
-  require_file "${RUNTIME_ENV_FILE}"
-  require_file "${SECRETS_ENV_FILE}"
-  rollback_to_previous_digest
-else
-  main
-fi
+[[ "$DEPLOY_MODE" == rollback ]] && manual_rollback || main
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)